src/legacy/core_plugins/data/public/query_bar/directive/index.js

```diff
@@ -21,18 +21,20 @@
 
 import 'ngreact';
 import { wrapInI18nContext } from 'ui/i18n';
-import { uiModules } from '../../modules';
+import { uiModules } from 'ui/modules';
 import { QueryBar } from '../components';
 
-const app = uiModules.get('app/kibana', ['react']);
+const app = uiModules.get('app/data', ['react']);
```

Comment:
I'm not sure it's worth investing time into angular code that will be removed soon anyway. What do you think?

Reply:
I guess what I meant was, will it make it easier to remove angular code later if it is scoped more granularly now, especially since we are already changing these names now? 

Definitely not worth investing the time if it won't help us later -- I was only suggesting it if you think it will save us time trying to figure out what's up with all of the various `app/data` references later.